examples/led_fade/README.md

```diff
@@ -0,0 +1,13 @@
+# LED Fade
+
+This project fades an LED when you turn a potentiometer knob.
+
+On the GrovePi+ or GrovePi Zero, connect a potentiometer to port A2 and a LED to port D3.
+
+If you're on Raspbian, trying this out by running:
```

Comment:
I like the idea of having nerves in an example. I personally have never run grovepi from anything but nerves. 

I think we can leave this one as a nerves example, but maybe we should name the directory `led_fade-nerves`?